var/spack/repos/builtin/packages/py-markdown2/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyMarkdown2(PythonPackage):
+    """A fast and complete Python implementation of Markdown."""
+
+    homepage = "https://github.com/trentm/python-markdown2"
+    pypi     = "markdown2/markdown2-2.3.9.tar.gz"
+
+    version('2.4.0',  sha256='28d769f0e544e6f68f684f01e9b186747b079a6927d9ca77ebc8c640a2829b1b')
+    version('2.3.9',  sha256='89526090907ae5ece66d783c434b35c29ee500c1986309e306ce2346273ada6a')
+
+    depends_on('python@2.6:2.8,3.3:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.5:3.999', when='@2.4.0:', type=('build', 'run'))
    depends_on('python@2.6:2.8,3.3:', when='@:2.3.999', type=('build', 'run'))
```

Reply:
Again, I've removed the older version of the package to keep things simple.